generators/app/templates/test/_library_/qunit/testsuite.qunit.js

```diff
@@ -20,7 +20,11 @@ sap.ui.define(function () {
 				qunitBridge: true,
 				useFakeTimers: false
 			},
-			module: "./{name}.qunit"
+			module: "./{name}.qunit",
+			coverage: {
+				only: ["<%= libURI %>"],
```

Comment:
```suggestion
				only: ["<%= libURI %>/"],
```
ending slash should be added

Reply:
done